js/battle.js

```diff
@@ -5331,6 +5331,7 @@ var Battle = (function () {
 		case 'c':
 			name = args[1];
 			if (this.ignoreSpects && (name.charAt(0) === ' ' || name.charAt(0) === '+')) break;
+			if (this.ignoreOpponent && this.getSide(name) === this.yourSide) break;
```

Comment:
I think the way it's currently written, it'd behave weirdly if the opponent changed their username (or username capitalization), and how do we want /addplayer to behave?